runtime/databricks/automl_runtime/forecast/prophet/model.py

```diff
@@ -117,35 +136,45 @@ def predict_timeseries(self, horizon: int = None) -> pd.DataFrame:
         """
         return self._predict_impl(horizon)
 
-    def predict(self, context: mlflow.pyfunc.model.PythonModelContext, input: pd.DataFrame) -> pd.DataFrame:
+    def predict(self, context: mlflow.pyfunc.model.PythonModelContext, input_df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
It doesn't seem we need `context` in this function?

Reply:
The context is required for mlflow.

Reply:
Can you tell me more about this context? The `mlflow.pyfunc.PyFuncModel` API does not seem to contain that part: https://github.com/mlflow/mlflow/blob/044ad9e545f4ee7ac23596a9a890bf5b30aac490/mlflow/pyfunc/__init__.py#L589

Reply:
It is trying to implement the function: https://github.com/mlflow/mlflow/blob/0fa849ad75e5733bf76cc14a4455657c5c32f107/mlflow/pyfunc/model.py#L86

Reply:
You may check the moflow doc for some examples: https://github.com/mlflow/mlflow/blob/19b3ad1009b668827a56be645caa5fd0280cab19/docs/source/models.rst#custom-python-models

Reply:
I see. Thanks for the pointer! One small nitpick: we should keep the parameter name same to the base class, i.e. `model_input` instead of `input_df` -- parameter name is part of the python API.